code/game/objects/items/theft_items.dm

```diff
@@ -202,13 +246,56 @@
 		if(ismob(loc))
 			to_chat(loc, "<span class='warning'>[src] is permanently sealed, [sliver] is safely contained.</span>")
 
+/obj/item/nuke_core_container/supermatter/unload(obj/item/retractor/supermatter/I, mob/user)
```

Comment:
(Not related to PR)


Its time to do surgery with supermatter tongs since its just a retractor.